README.md

```diff
@@ -160,6 +160,11 @@ The map of containers consists of the name of the container mapped to the contai
 * `build` (object, optional): Parameters mapped to Docker's `build`.
 	* `context` (string)
 	* `file` (string)
+* `push` (object, optional): Parameters related to Docker's `push`.
+  * `skip` (boolean): prevents pushing image when `push` is called
```

Comment:
You mean CLI indication instead of `crane.yml` ? This sounds like a hassle to build a proper command and will hide the actual behavior when developers are just looking at the `yml` file.